manifests/freshclam.pp

```diff
@@ -23,13 +24,15 @@
   $proxy_username = hiera('clamav::freshclam::proxy_username',''),
   $proxy_password = hiera('clamav::freshclam::proxy_password',''),
   $logfile = '/var/log/clamav/freshclam.log',
+  $config = hiera('clamav::freshclam::config','/etc/freshclam.conf')
 ) {
   include clamav::params
 
-  file { '/etc/freshclam.conf':
+  file { $config:
     ensure  => present,
     owner   => $clamav::params::user,
-    mode    => '0400',
+    group   => root,
+    mode    => '0440',
```

Comment:
suggest parameterizing `mode` as well since not everyone wants it enforced this strictly.